cves/CVE-2011-3923.yml

```diff
@@ -90,15 +81,13 @@ previous_fixes:
 future_fixes:
   - commit:
     note:
-
 upvotes_instructions: |
   Students: when initially writing this, ignore this upvotes number.
   Once this work is being reviewed, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
 upvotes:
```

Comment:
3 upvotes

Reply:
Thanks for the upvotes! I forgot to add some to yours so I will do so shortly!